rcl_action/test/rcl_action/test_graph.cpp

```diff
@@ -132,17 +133,14 @@ TEST_F(
   rcl_reset_error();
   // Invalid node name
   ret = rcl_action_get_client_names_and_types_by_node(
-    &this->node, &this->allocator, "_test_this_Isnot_a_valid_name", "", &nat);
-  EXPECT_EQ(RCL_RET_ERROR, ret) << rcl_get_error_string().str;
+    &this->node, &this->allocator, "_!test_this_is_not_a_valid_name", "", &nat);
```

Comment:
Is the previous value , "_test_this_Isnot_a_valid_name", now valid?

Reply:
Yes, it was always valid.
Before, the error code for invalid node name and for nonexistent nodes was the same, so the test was passing.
But, AFAIK, caps were always accepted as topic name: https://github.com/ros2/design/pull/241.

Reply:
Ah, I see. Thanks.